RealtekAC97.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = Realtek Audio Drivers for AC97 A4.06
+Version = A4.06 (5.10.0.6300)
+License = Propietary
+Description = Audio driver for Win98/Win2000/WinME/WinXP (WDM driver) for multiple AC97 cards like ALC850, ALC655 or ALC250. Suitable for multiple old workstation systems with VIA or C-media chipsets.
+Category = 13
+URLSite = https://www.realtek.com/
+URLDownload = https://akacc.net/win98/0001-WDM_A406.exe
```

Comment:
Does the URLSite links to the specified download URL?

Reply:
Actually, we can't have a functional download link from the Realtek Website because it needs a captcha and a tocken every single time you try to download the drivers. So... In that side, we are a bit "blocked" to access the original source, we need that alternative or some "webarchive" link.